internal/tsoptions/tsconfigparsing_test.go

```diff
@@ -471,6 +471,19 @@ var parseJsonConfigFileTests = []struct {
 			allFileList:    map[string]string{"/tsconfig.base.json": tsconfigWithExtendsAndConfigDir, "/src/index.ts": "", "/src/app.ts": "", "/node_modules/module.ts": "", "/dist/output.js": ""},
 		}},
 	},
+	{
```

Comment:
I am not really sure if I understand how the test suite works correctly and if I added the test case to the right place. :thinking:

Reply:
I think this is right...